### PR TITLE
SDK & adapters merge

### PR DESCRIPTION
**Pull request by gaya3-zipstack**
## What

Merge adapters into SDK. Consider this as a refactoring PR with the scope of  bringing the existing adapters code into SDK under a new folder called "adapters"

## Why

To reduce the repositories to be maintained. SDK and adapters look hand in hand for now and so easy cutting down on one repository helps developer time spent on maintaining versions of adapters.

## How

Bring existing adapters code into SDK into a new folder called "adapters"
All existing imports that look like "unstract.adapters." will now be "unstract.sdk.adapters"
Imports of the sdk "unstract.sdk.*" does not have any change as the nesting is kept the same.

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions / Env Variables

-

## Notes on Testing

Needed to be integrated into unstract repo

## Screenshots

...

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: pyproject.toml
Comment (gaya3-zipstack) [inline pyproject.toml:1]: Merge adapters pyproject.toml into sdk. WOndering if we can add a new group "adapters" and get this in @chandrasekharan-zipstack ? Will that work?

Comment (chandrasekharan-zipstack) [reply pyproject.toml:1]: I think new groups are meant to be optional dependencies. Just doing a `pdm install` would install all groups. However when we do a `pdm sync --prod` we might have to specify the optional groups also explicitly in our Dockerfile. If you think we'll use the SDK alone without the adapters in any of the services, we could do this to reduce the package size. If we do adopt specifying the adapter dependencies as an optional group - we could split it as `llm`, `vector_db` and so on to allow using a type of adapter alone for a service.

File: src/unstract/sdk/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = "0.38.0"
+__version__ = "0.40.8"
```
[the remark this answers is not in the document]
Comment (gaya3-zipstack) [reply src/unstract/sdk/__init__.py:1]: This is dummy for  now. Anyway will have to roll based on latest. Will do that. For now, pls ignore this.

File: pyproject.toml
Comment (gaya3-zipstack) [reply pyproject.toml:1]: Got it. Will look into it later for improvements. Thanks @chandrasekharan-zipstack

File: src/unstract/sdk/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (gaya3-zipstack) [reply src/unstract/sdk/__init__.py]: Merged latest and made this 0.39.0.

**Review by chandrasekharan-zipstack — APPROVED** [review]

**Review by Deepak-Kesavan — APPROVED** [review]

**Review by chandrasekharan-zipstack — APPROVED** [review]

**Pull request merged**